install_config/configuring_authentication.adoc

```diff
@@ -159,18 +159,24 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Configuring the ldap ca certificate <1>
 #openshift_master_ldap_ca=<ca text>
 # or
-#openshift_master_ldap_ca_file=<path to local ca file to use>
+#openshift_master_ldap_ca_file=<path to local ca file to use> <2>
 
 # Available variables for configuring certificates for other identity providers:
 #openshift_master_openid_ca
-#openshift_master_openid_ca_file
+#openshift_master_openid_ca_file <2>
 #openshift_master_request_header_ca
-#openshift_master_request_header_ca_file
+#openshift_master_request_header_ca_file <2>
 ----
 <1> If you specify your CA certificate location in the
 `openshift_master_identity_providers` parameter, do not specify a certificate
 value in the `openshift_master_ldap_ca` parameter or path in the
 `openshift_master_ldap_ca_file` parameter.
+<2> If you specify a file on the host you run the playbook on, its contents are
+copied to the *_/etc/origin/master/<identity_provider_name>_ca.crt_* file. The
+identity provider name is `ldap`, `openid`, or `request_header` to match the
```

Comment:
```suggestion
 identity provider name is name of the item in `openshift_master_identity_providers` list and identity provider type is `ldap`, `openid`, or `request_header` to match the
```

In `openshift_master_identity_providers` two or more identity providers can be specified:
```
openshift_master_identity_providers:
- name: foo
  provider:
    kind: OpenIDIdentityProvider
- name: bar
  provider:
    kind: OpenIDIdentityProvider
- name: baz
  provider:
    kind: RequestHeaderIdentityProvider
```
This config would require the following files:
* `/etc/origin/master/foo_openid_ca.crt`
* `/etc/origin/master/bar_openid_ca.crt`
* `/etc/origin/master/baz_requestheader_ca.crt`

Reply:
@enj, are you ready for us to tell people that you can use multiple identity providers?

Reply:
Mo says that you've always been able to do this.